MdeModulePkg/Bus/Pci/NvmExpressDxe/NvmExpressPassthru.c

```diff
@@ -824,6 +824,8 @@ NvmExpressPassThru (
     //
     CopyMem (Packet->NvmeCompletion, (VOID *)Cq, sizeof (EFI_NVM_EXPRESS_COMPLETION));
   } else {
+    ReportStatusCode ((EFI_ERROR_MAJOR | EFI_ERROR_CODE), (EFI_IO_BUS_SCSI | EFI_IOB_EC_INTERFACE_ERROR));

```

Comment:
According to current logic, EFI_IOB_EC_INTERFACE_ERROR is used when the device error. It is not specific for time out.

Reply:
Gentle ping @lgao4 .